pyccel/parser/syntactic.py

```diff
@@ -643,6 +647,21 @@ def fill_types(ls):
             return container
 
         decorators = {}
+
+        # add the decorator @types if the arguments are annotated
```

Comment:
If the user has provided a decorator and adds type annotations do we get a warning? Should we?

Reply:
it will be an interface, if it's duplicated we will get a warning